### ISSUE TITLE
TypeError: wikibase-form claim parser isn't implemented

### ISSUE THREAD

**Issue by bsmcgillis** [maxlath/wikibase-sdk#67]
I'm encountering this error just a couple of minutes after the parser begins running. Strangely, I haven't changed anything with my set up. And I've tried to run the same parse in two different environments and both encounter the same issue. 

Claim ID: Q275937$60A491E6-E5FF-4A2E-934C-9F0FD3072539

Stack Trace:
```
at parse (/usr/local/lib/node_modules/wikidata-filter/node_modules/wikidata-sdk/lib/helpers/parse_claim.js:93:31)
    at simplifyClaim (/usr/local/lib/node_modules/wikidata-filter/node_modules/wikidata-sdk/lib/helpers/simplify_claims.js:96:13)
    at propClaims.map.claim (/usr/local/lib/node_modules/wikidata-filter/node_modules/wikidata-sdk/lib/helpers/simplify_claims.js:33:19)
    at Array.map (<anonymous>)
    at simplifyPropertyClaims (/usr/local/lib/node_modules/wikidata-filter/node_modules/wikidata-sdk/lib/helpers/simplify_claims.js:33:6)
    at simplifyClaims (/usr/local/lib/node_modules/wikidata-filter/node_modules/wikidata-sdk/lib/helpers/simplify_claims.js:14:24)
    at Object.simplifyEntity [as entity] (/usr/local/lib/node_modules/wikidata-filter/node_modules/wikidata-sdk/lib/helpers/simplify_entity.js:17:25)
    at module.exports (/usr/local/lib/node_modules/wikidata-filter/lib/format_entity.js:24:27)
    at wdFilter (/usr/local/lib/node_modules/wikidata-filter/lib/wikidata_filter.js:28:12)
    at Stream.<anonymous> (/usr/local/lib/node_modules/wikidata-filter/lib/filter.js:7:19)
```

And here is the command I'm running: 
```
DUMP='https://dumps.wikimedia.org/wikidatawiki/entities/latest-all.json.gz'
curl $DUMP | gzip -d | wikidata-filter --claim P31:Q5 --sitelink enwiki --keep id,labels,sitelinks,claims --languages en --simplify '{}' > humans_2.ndjson
```

**Comment by maxlath** [comment]
thanks for reporting, fixed by 4e7fda5 and published in `v7.3.1`

**Comment by maxlath** [comment]
I updated wikidata-filter wikidata-sdk version to include this patch: `wikidata-filter@4.0.2`
